package-lock.json

```diff
@@ -7708,7 +7708,7 @@
 							"dependencies": {
 								"minimist": {
 									"version": "1.2.0",
-									"resolved": false,
+									"resolved": "",
```

Comment:
This is surprising to see those changes.

Reply:
Oh bugger, shouldn't have committed this file. I keep getting this diff when I run `npm install` even though I'm updating my npm version regularly 😩

Reply:
So reverting this file to its previous state before my commit doesn't work because it's been updated in master in the meantime. I tried `nvm install --latest-npm ` followed by `npm install` and even `npm run docs:build` on my local branch and there's no diff. Is it possible the current `package-lock` on master wasn't generated with the latest npm?

Reply:
Yes, if there is new npm version, and they changed something then we will have to patch `master`.